test/library/run-options.test.js

```diff
@@ -239,4 +241,66 @@ describe('Newman run options', function () {
             });
         });
     });
+
```

Comment:
- add test for default behavior, without `reporters` and `executionsSummary` option
- add a test with `reporters: 'cli'`